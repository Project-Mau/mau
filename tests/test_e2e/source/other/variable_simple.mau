:answer:42

The Answer to the Ultimate Question of Life, the Universe, and Everything is {answer}.

Interpolation can be blocked escaping the curly braces: \{answer}. This happens automatically in verbatim: `{answer}`.
