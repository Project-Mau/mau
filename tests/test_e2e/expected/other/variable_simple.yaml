data:
  args: []
  content:
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: The Answer to the Ultimate Question of Life, the Universe, and Everything
            is 42.
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: 'Interpolation can be blocked escaping the curly braces: {answer}.
            This happens automatically in verbatim: '
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: verbatim
          value: '{answer}'
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: .
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  kwargs: {}
  subtype: null
  tags: []
  type: document

